pkg/table/column_test.go

```diff
@@ -412,8 +419,9 @@ func TestGetDefaultValue(t *testing.T) {
 		{
 			&model.ColumnInfo{
 				FieldType:          tp3,
-				OriginDefaultValue: timestampValue.String(),
-				DefaultValue:       timestampValue.String(),
+				OriginDefaultValue: timestampValueUTC.String(),
+				DefaultValue:       timestampValueUTC.String(),
+				Version:            model.ColumnInfoVersion2,
```

Comment:
As the comments:

https://github.com/pingcap/tidb/blob/e528b6819e0969b415d5db746473b31f6edb5773/pkg/parser/model/model.go#L123-L129


We should set the version to 2 and use the UTC as the timezone for the timestamp field's default value